tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -1,4 +1,5 @@
 def improper_indents(indents):
+
```

Comment:
I think what this merge should (and does) bring to the table is the ability of the bear to actually check and fix paddings both on the top and the bottom, depending on what the style demands. The actual issue that this now violates PEP257 lies in the style definition [here](https://github.com/coala/coala/pull/4557/files#diff-eb7df11f93546c67d1807d6649d54216) <- this is wrong and hence the files in this PR are wrong. 

You should probably write tests that don't rely on the styles defined in coala, but rather use ad-hoc styles. It will make it clearer what happened anyway.